source/documentation/deploying-an-app/multicontainer-app-deploy.md

```diff
@@ -100,3 +122,10 @@ It should behave in the same way as when you were running it locally via docker-
 [ecr-setup]: tasks.html#creating-an-ecr-repository
 [add-aws-resources]: tasks.html#adding-aws-resources-to-your-environment
 [rds-access-instructions]: https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance#access-outside-the-cluster
+[creating-dashboards]: https://user-guide.cloud-platform.service.justice.gov.uk/tasks.html#creating-dashboards
```

Comment:
Please change the absolute URLs which point to other parts of the user guide to be relative URLs (i.e. remove "https://user-guide.cloud-platform.service.justice.gov.uk/")

Reply:
Thanks @digitalronin updated the URL